example/example.cpp

```diff
@@ -109,21 +109,21 @@ void stdout_logger_example()
 void basic_example()
 {
     // Create basic file logger (not rotated).
-    auto my_logger = spdlog::basic_logger_mt("file_logger", "logs/basic-log.txt");
+    auto my_logger = spdlog::basic_logger_mt("file_logger", SPDLOG_FILENAME_T("logs/basic-log.txt"));
```

Comment:
While correct, I think the example should be without this SPDLOG_FILENAME_T macro. It is only needed in very special cases (windows + wchar support).

Reply:
The CI builds the example also, which means that the entry I added to the AppVeyor matrix fails without this.

Reply:
Yes, but for such specific case that i must insist. Otherwise users might think this ugly macro is the standard usage

Reply:
Resolved